Example/app.js

```diff
@@ -6,77 +6,97 @@
  *
  */
 var TiIdentity = require('ti.identity');
-
+var isandroid = Ti.Platform.name === 'android';
 var isiOS = Ti.Platform.name === 'iOS' || Ti.Platform.name === 'iPhone OS';
 var authPhrase = 'Fingerprint';
+var supported =  TiIdentity.isSupported();
 
-// You can set the authentication policy on iOS (biometric or passcode)
-// In this case, we also check for Touch ID vs Face ID for a more personalized UI
-if (isiOS) {
-	// In order to check the biometry type, you have to check if biometrics are supported in general
-	var supported =  TiIdentity.isSupported();
-	
-	if (TiIdentity.biometryType == TiIdentity.BIOMETRY_TYPE_FACE_ID) {
-		authPhrase = 'Face ID';
-	} else if (TiIdentity.biometryType == TiIdentity.BIOMETRY_TYPE_TOUCH_ID) {
+var win = Ti.UI.createWindow();
+
+// Using this constant, ios will offer to authenticate with Touch ID and Face ID and android will
+// offer to authenticate with Touch ID.
+// When calling "authenticate" below
+TiIdentity.setAuthenticationPolicy(TiIdentity.AUTHENTICATION_POLICY_BIOMETRICS);
+
+if (isandroid) {
+
+	if (TiIdentity.getAuthenticationPolicy() === TiIdentity.AUTHENTICATION_POLICY_BIOMETRICS) {
 		authPhrase = 'Touch ID';
+	} else if (TiIdentity.getAuthenticationPolicy() === TiIdentity.AUTHENTICATION_POLICY_PASSCODE) {
+		authPhrase = 'Device Passcode';
 	} else {
-		authPhrase = '(None available)'
+		authPhrase = '(None available)';
 	}
+}
 
-	if (!supported) {
-		alert('Authentication is not supported. Available biometrics: ' + authPhrase);
+if (isiOS) {
+	if (TiIdentity.biometryType === TiIdentity.BIOMETRY_TYPE_FACE_ID) {
+		authPhrase = 'Face ID';
+	} else if (TiIdentity.biometryType === TiIdentity.BIOMETRY_TYPE_TOUCH_ID) {
+		authPhrase = 'Touch ID';
+	} else {
+		authPhrase = '(None available)';
```

Comment:
the none available constant can be created into a string constant variable and used for both android & ios check.

Reply:
Done